service/src/test/java/com/epam/esm/service/impl/OrderServiceImplTest.java

```diff
@@ -102,7 +102,7 @@ void shouldDelegateFindByIdToUserDaoThenFindByUserAndIdToOrderDaoAndMappingToMap
         LocalDateTime now = LocalDateTime.now();
```

Comment:
<img width="908" alt="image" src="https://user-images.githubusercontent.com/17727888/159156948-e913eec6-1c08-4f77-b570-2932c1c6cb21.png">
EngX Bootcamp, Unit testing module

Reply:
You can have at least have a shared class that will provide prebuilt Orders, Users, etc, so it will improve readability.